mesa/visualization/mpl_space_drawing.py

```diff
@@ -108,7 +108,9 @@ def draw_space(
     space,
     agent_portrayal: Callable,
     propertylayer_portrayal: dict | None = None,
-    ax: Axes | None = None,
+    ax_grid: Axes | None = None,
```

Comment:
I am not convinced that this is the ideal solution for an improved API. You now have to pass the axes multiple times in the most common use case. I suggest we hatch out the API design in #2640 first before opening a PR.